### DISCUSSION TITLE
ccf-validation-tools/Ureter/comments/

### DISCUSSION THREAD

**Discussion by giscus** [hubmapconsortium/ccf-validation-tools#278] [General]
# ccf-validation-tools/Ureter/comments/

https://hubmapconsortium.github.io/ccf-validation-tools/Ureter/comments/

**Comment by emquardokus** [comment]
Ureter_v1.1_DRAFT
https://docs.google.com/spreadsheets/d/1ZUmHX22NYMfBgFoni4zK6bsEYFn4rGSk9oYBNPcebZQ/edit#gid=73126811&range=15:15

**Blank ontology ID missing parent**
ADDED CT/1 ureter smooth muscle cell CL:1000979    In row 15, the term detrusor smooth muscle cell of ureter without ontology ID has no parent that is from the CL ontology.

**Comment by aleixpuigb** [comment]
**New anatomical structures added:**

Ureter : proximal ureter (UBERON:8600108)
Ureter : distal ureter (UBERON:8600109)
Ureter : intramural ureter (UBERON:8600110)
Ureter : right distal ureter (UBERON:8600111)
Ureter : left distal ureter (UBERON:8600112)
Ureter : right proximal ureter (UBERON:8600113)
Ureter : left proximal ureter (UBERON:8600114)
Ureter : right intramural ureter (UBERON:8600115)
Ureter : left intramural ureter (UBERON:8600116)
